apps/docs/src/app/core/component-docs/multi-input/examples/multi-input-form-example/multi-input-form-example.component.html

```diff
@@ -12,7 +12,6 @@
             <fd-multi-input
                 formControlName="disabledSelectedValues"
                 [dropdownValues]="['Apple', 'Banana', 'Pineapple', 'Tomato']"
-                [disabled]="'true'"
```

Comment:
Disabled state is added in typescript file. It was causing some console warnings.